src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/scenario_page/scenario_page.py

```diff
@@ -1,8 +1,17 @@
-from taipy.gui import Markdown
+from taipy.gui import Markdown, notify
 
 from .data_node_management import manage_partial
 
 
+def notify_on_submission(state, submitable, details):
+    if details['submission_status'] == 'COMPLETED':
+        state.refresh('selected_scenario')
```

Comment:
What is this refresh for?

Reply:
For all the expressions that have in them 'selected_scenario'

Reply:
Here, there are no such expressions. Do you want to delete this line?

Reply:
I would say yes, if it s not used.